reconstruction.py

```diff
@@ -1,5 +1,3 @@
-"""Reconstruct 3D transformation of cameras.
```

Comment:
somehow this comment got lost in the merge lol maybe just put it back in before we merge